modules/rbenv/rbenv.nu

```diff
@@ -8,23 +8,19 @@ export-env {
     }
 }
 
-export def-env rbenv [
-	command?: string@'nu-complete rbenv',
-	...args
-] {
+def rbenv [command?: string@'nu-complete rbenv', ...args] {
```

Comment:
if you want to `use rbenv.nu` and then be able to run `rbenv` this won't work :thinking: 

i think you'll either have to
- keep that but do `source rbenv.nu` instead of `use`
- revert this PR to `export`

also it looks like this command tries to mutate the environment with `load-env`, i think you cannot remove the `dev-env`

Reply:
Thanks for the review